traffic_ops/traffic_ops_golang/vault/bucket.go

```diff
@@ -22,13 +22,22 @@ package apiriak
 import (
 	"encoding/json"
 	"errors"
-	"net/http"
-
+	"github.com/apache/trafficcontrol/lib/go-tc"
+	"github.com/apache/trafficcontrol/lib/go-util"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/riaksvc"
+	"net/http"
 )
 
+func GetBucketKeyDeprecated(w http.ResponseWriter, r *http.Request) {
+	getBucketKey(w, r, api.CreateDeprecationAlerts(util.StrPtr("/value/bucket/:bucket/key/:key/values")))
```

Comment:
Should be `/vault/bucket/:bucket/key/:key/values`, not `/value/bucket/:bucket/key/:key/values`.